frontend/src/features/projects/list/SPLProjectListView.test.tsx

```diff
@@ -0,0 +1,174 @@
+      subAgency: 'UOV',
+      properties: [
+        {
+          id: 1,
+          propertyTypeId: 1,
+          description:
+            'Gym, teaching kitchen, energy lab, greenhouse. Building full name: Center for Excellence in Sustainability',
+          classification: 'Core Operational',
+          agencyId: 0,
+          subAgency: 'Nichola Valley Institute of Technology',
+          agencyCode: 'AEST',
+          address: '4155 Belshaw St',
+          city: 'Merritt',
+          netBook: 0,
+          assessed: 0,
+          market: 0,
+          landArea: 26.9,
+          createdOn: '2020-05-28T18:48:03.181977',
+          rowVersion: 'AAAAAAAAfRA=',
+        },
+      ],
+      createdBy: 'User, Administrator',
+      createdOn: '2020-05-28T18:48:03.181976',
+      rowVersion: 'AAAAAAAAfQ8=',
+    },
+    {
+      projectNumber: 'SPP-10016',
+      name: 'Project name 17',
+      statusId: 40,
+      status: 'Pre-Marketing',
+      tierLevelId: 1,
+      tierLevel: 'Tier 1',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+      note: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+      agencyId: 35,
+      agency: 'AEST',
+      subAgency: 'UOV',
+      properties: [
+        {
+          id: 1,
+          propertyTypeId: 1,
+          description:
+            'Gym, teaching kitchen, energy lab, greenhouse. Building full name: Center for Excellence in Sustainability',
+          classification: 'Core Operational',
+          agencyId: 0,
+          subAgency: 'Nichola Valley Institute of Technology',
+          agencyCode: 'AEST',
+          address: '53033 Lake Rd.',
+          city: 'Chilliwack',
+          netBook: 0,
+          assessed: 0,
+          market: 0,
+          landArea: 26.9,
+          createdOn: '2020-05-28T18:48:03.181977',
+          rowVersion: 'AAAAAAAAfRA=',
+        },
+      ],
+      createdBy: 'User, Administrator',
+      createdOn: '2020-05-28T18:48:03.181976',
+      rowVersion: 'AAAAAAAAfQ8=',
+    },
+  ],
+  page: 1,
+  quantity: 10,
+  pageIndex: 0,
+  total: 1,
+};
+
+// Set all module functions to jest.fn
+jest.mock('../apiService');
+const mockedService = service as jest.Mocked<typeof service>;
+
+const mockStore = configureMockStore([thunk]);
+
+const lCodes = {
+  lookupCodes: [
+    { name: 'agencyVal', id: '1', isDisabled: false, type: API.AGENCY_CODE_SET_NAME },
+    {
+      name: 'classificationVal',
+      id: '1',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+  ] as ILookupCode[],
+};
+
+const store = mockStore({
+  [reducerTypes.LOOKUP_CODE]: lCodes,
+});
+
+const history = createMemoryHistory();
+
+describe('Project Approval Request list view', () => {
+  // clear mocks before each test
+  beforeEach(() => {
+    mockedService.getProjectList.mockClear();
+    mockedService.deleteProject.mockClear();
+  });
+
+  afterEach(() => {
+    cleanup();
```

Comment:
Does this also clear the other mocks in this file like the `apiService`?

Reply:
I'm not sure, how can I test that it does that?

Reply:
a question for Devin, perhaps, as I basically copied the ProjectApprovalRequest.test.tsx file.

Reply:
Should we add `mockedService.getProjectList.mockClear();
    mockedService.deleteProject.mockClear();` after the cleanup() in the afterEach() method?

Reply:
@devinleighsmith Can you provide insight here?